lib/grpc_or_grpcweb.dart

```diff
@@ -17,9 +17,9 @@
 /// on all platfroms except web, on which it uses [GrpcWebClientChannel].
 ///
 /// Note that gRPC and gRPC-web are 2 different protocols and server must be
```

Comment:
Consider removing "usually on separate ports". Maybe: 

```dart
// We allow to specify two different ports in case
// if the same port does not speak both protocols. 
```

Reply:
I've reworded this source doc completely: please have a look.